testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/IdentityProviderTest.java

```diff
@@ -608,6 +609,11 @@ public void testSamlImportAndExport() throws URISyntaxException, IOException, Pa
         form.addFormData("file", body, MediaType.APPLICATION_XML_TYPE, "saml-idp-metadata.xml");
 
         Map<String, String> result = realm.identityProviders().importFrom(form);
+       //firtsly check and remove enabled from result
+        boolean enabled = Boolean.valueOf(result.get("enabled"));
+        result.remove("enabled");
```

Comment:
Use constant, not literal: 
```suggestion
        boolean enabled = Boolean.valueOf(result.get(ENABLED_FROM_METADATA));
        result.remove(ENABLED_FROM_METADATA);
```

Reply:
I strongly agree with you. However, in class IdentityProviderTest only literals are used and I want class writing to be similar.
If you insist, I could add literal for this or for all config properties.

Reply:
Thank you. Constant just for this single change would set a good example for the rest, but those needn't be addressed in this PR.